software/debugging/index.rst

```diff
@@ -0,0 +1,456 @@
+.. _debugging:
```

Comment:
This tag is already used in the summit guide (line 2473). So maybe call this tag "software-debugging" ? It's either that or change the Summit one. But other things may break if you change the Summit one because of how long it has been around (unsure what links to it).

Reply:
fixed in 15e4ea35ac8e6c884ee1e03be22a3c76c1d214d3